README.md

```diff
@@ -32,17 +32,33 @@ Also, install some thermostat dependencies.
 sudo apt install sqlite3
 ```
 
-Compile and install libegt to a directory. We'll assume it's installed to
-/opt/egt. Then, configure and build this project.
+Compile and install libegt to a directory. Then, configure
 
 ```sh
 git clone --recursive https://github.com/linux4sam/egt-thermostat.git
 cd egt-thermostat
 ./autogen.sh
+```
+
+Depending on your egt path the next step could be:
+```sh
 CXXFLAGS="-I/opt/egt/include/" \
 LDFLAGS="-L/opt/egt/lib/" \
 PKG_CONFIG_PATH=/opt/egt/lib/pkgconfig \
 ./configure
+```
+
+**or**
+
+```sh
+CXXFLAGS="-I/usr/local/include/" \
+LDFLAGS="-L/usr/local/lib/" \
+PKG_CONFIG_PATH=/usr/local/lib/pkgconfig \
+./configure
+```
+
+& then build this project
+```sh
```

Comment:
Honestly I don't see the benefit to add this for several reasons:
- At the beginning, we state egt is installed to /opt/egt so it's obvious that if it's not the case the path has to be updated.
- This add an extra case but there are other ones.
- With a distribution as Ubuntu, this is the default path so it's not needed, you can just do ./configure without setting any variable.